Ports/lxt/patches/0001-Add-support-for-SerenityOS.patch

```diff
@@ -0,0 +1,59 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Seal Sealy <butilkama@gmail.com>
+Date: Wed, 15 Mar 2023 17:24:38 +0300
+Subject: [PATCH] Add support for SerenityOS
+ This patch adds support for SerenityOS.
+ Specifially, it does the following things:
+ 1. Specifies MAXNAMLEN, otherwise it's undefined and build fails
+ 2. Addresses issues with configure.ac
+ 3. Fixes path to clear utility
+
+---
+ compatible.h | 2 +-
+ configure.ac | 3 ---
+ main.c       | 2 +-
+ 3 files changed, 2 insertions(+), 5 deletions(-)
+
+diff --git a/compatible.h b/compatible.h
+index ce045d6badfa7f580867a964001709f2a25bc430..d39ca70693a06ed81fa40d4454640426f109d8f3 100644
+--- a/compatible.h
++++ b/compatible.h
+@@ -7,7 +7,7 @@
+ 	unpredictible things when you deal with paths that are longer than this
+ 	number. */
+ /* #define PATH_MAX 4096 */  /* length of longest pathname */
+-/* #define MAXNAMLEN 256 */  /* length of longest filename */
++#define MAXNAMLEN 255  /* length of longest filename */
```

Comment:
Is this something that is somewhat standardized? If so, we could just add it to the system properly instead of patching it in here.

Reply:
While it's not a part of SUSv2 it's used in glibc, FreeBSD and Illumos (OpenSolaris) so I assume it's somewhat standardized. I'll push out a commit including that to where it's supposed to be.